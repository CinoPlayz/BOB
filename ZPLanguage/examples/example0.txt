infrastructure "Celje-Maribor" {
    let mariborPostaja = station "Maribor" {
        box((6+2,4), (-63,3));
        platform 1, 2 {
            box((5,2), (4,3));
        }
    }
}