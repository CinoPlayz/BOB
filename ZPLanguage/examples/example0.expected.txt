{
                  "type": "FeatureCollection",
                  "features": [                {
                  "type": "Feature",
                  "properties": {
                    "type": "station",
                    "name": "Maribor"
                  },
                                      "geometry": {
   "type": "Polygon",
   "coordinates": [[
      [4.0, 8.0],
      [3.0, 8.0],
      [3.0, -63.0],
      [4.0, -63.0],
      [4.0, 8.0]
   ]]
}
                },                {
                  "type": "Feature",
                  "properties": {
                    "type": "platform",
                    "number": "1",
                    "countOfTracks": "2",
                    "stationName": "Maribor"
                  },
                                      "geometry": {
   "type": "Polygon",
   "coordinates": [[
      [2.0, 5.0],
      [3.0, 5.0],
      [3.0, 4.0],
      [2.0, 4.0],
      [2.0, 5.0]
   ]]
}
                }],
                  "metadata": {
                    "name": "Celje-Maribor"
                  }
                }